packages/components/date-picker/src/date-picker-com/basic-date-table.vue

```diff
@@ -389,7 +373,7 @@ export default defineComponent({
 
     const isWeekActive = (cell) => {
       if (props.selectionMode !== 'week') return false
-      let newDate = props.date.startOf('day')
+      let newDate = props.date?.startOf('day')!
```

Comment:
I think this is somehow conflict since `date` is marked as  `optional` then you get the `!` mark for `startOf` method, question is why don't we do
```ts
      let newDate = props.date!.startOf('day')
```
instead?

Reply:
OK, I'll replace it.

Reply:
OK, I'll replace it.